integration/xgboost/encryption_plugins/tests/test_tenseal.py

```diff
@@ -95,12 +96,12 @@ def test_hori() -> None:
         out_len1 = ctypes.c_size_t()
 
         nvflare.FederatedPluginSyncEnrcyptedHistHori(
-                handle,
-                out,
-                out_len,
-                ctypes.byref(out1),
-                ctypes.byref(out_len1),
-            )
+            handle,
```

Comment:
Style fixes in other places are great, but please move these changes into another PR.